project-base/storefront/components/Pages/ProductComparison/ProductComparison.tsx

```diff
@@ -1,40 +1,40 @@
 import { ProductComparisonContent } from './ProductComparisonContent';
 import { Heading } from 'components/Basic/Heading/Heading';
 import { InfoIcon } from 'components/Basic/Icon/IconsSvg';
-import { Loader } from 'components/Basic/Loader/Loader';
+import { SkeletonModuleComparison } from 'components/Blocks/Skeleton/SkeletonModuleComparison';
 import { Webline } from 'components/Layout/Webline/Webline';
 import { useGtmSliderProductListViewEvent } from 'gtm/hooks/productList/useGtmSliderProductListViewEvent';
 import { GtmProductListNameType } from 'gtm/types/enums';
+import { useIsUserLoggedIn } from 'hooks/auth/useIsUserLoggedIn';
 import { useComparison } from 'hooks/comparison/useComparison';
 import useTranslation from 'next-translate/useTranslation';
+import { usePersistStore } from 'store/usePersistStore';
 
 export const ProductComparison: FC = () => {
     const { t } = useTranslation();
 
     const { comparison, fetching } = useComparison();
 
-    const content = comparison?.products ? (
-        <ProductComparisonContent productsCompare={comparison.products} />
-    ) : (
-        <div className="my-[75px] flex items-center">
-            <InfoIcon className="mr-4 w-8" />
-
-            <Heading className="!mb-0" type="h3">
-                {t('Comparison does not contain any products yet.')}
-            </Heading>
-        </div>
-    );
-
     useGtmSliderProductListViewEvent(comparison?.products, GtmProductListNameType.product_comparison_page);
 
     return (
         <Webline>
-            {fetching ? (
-                <div className="flex w-full justify-center py-10">
-                    <Loader className="w-10" />
+            <Heading className="flex w-full items-end lg:mb-8 lg:w-auto lg:flex-1" type="h1">
+                {`${t('Product comparison')}${comparison?.products.length ? ` (${comparison.products.length})` : ''}`}
+            </Heading>
+
+            {fetching && <SkeletonModuleComparison />}
+
+            {!!comparison?.products && !fetching && <ProductComparisonContent productsCompare={comparison.products} />}
```

Comment:
Cannot these two be joined into a ternary operator? Why does one condition have `!!comparison?.products` and the second does not?